packages/react/src/Grid/Grid.test.tsx

```diff
@@ -87,8 +88,23 @@ describe('Grid', () => {
     })
   })
 
+  gridTags.forEach((tag) => {
+    it(`renders with a custom ${tag} tag`, () => {
+      const { container } = render(<Grid as={tag} aria-label={tag === 'section' ? 'Accessible name' : undefined} />)
```

Comment:
Is the `aria-label` necessary?

Reply:
It allows selecting by role below, for all tags except for the `div`. Would you prefer removing `aria-label` and selecting by `:only-child`?

Reply:
Ah, right. No, selecting by role has preference